src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +28,32 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o instanceof Rectangle) {
+            Rectangle rectangle = (Rectangle) o;
+            if (!this.width.equals(rectangle.width)) {
+                return false;
+            }
+            if (!this.length.equals(rectangle.length)) {
```

Comment:
@Rosinskyi simplify this if statements

Reply:
collapse them

Reply:
if i write like `if (!this.width.equals(rectangle.width)) return false;` i'll have checkstyle error 'if' construct must use '{}'s. (40:0) [NeedBraces]